parameter_files/archive/api27.0.0_062823_pr1093.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<all>
+  <!-- This parameter patch file updates the parameter file with the necessary changes -->
+  <!-- for PR  1093. This PR updates the allometry parameters for the default global PFTs -->
+
+
+  <!-- General xml file usage notes: -->
+  <!-- Special mod types "variable_add": creates a new variable
+		 "dimension_add": creates a new dimension
+		 "variable_del": removes a variable, must include attribute "name" 
+		 "variable_change": change values in the parameter, possibly including the name, and
+                          this must also come with a name attribute as well -->
+  <!-- To change a dimension. The easiest way to script this, is to delete the variables
+		 that use it, delete the dimension. Re-add it with the new size, and then re-add
+		 the variables -->
+  <!-- -->
+  
+  <base_file>archive/api27.0.0_062823_fates_params_default.cdl </base_file>
+  <new_file>fates_params_default.cdl</new_file>
+  <pft_list> 1,2,3,4,5,6,7,8,9,10,11,12 </pft_list>
+  <mods>
+    <mod type="variable_change" name="fates_allom_agb1">
+      <di>  fates_pft </di>
```

Comment:
Not sure if we have  a precedent of this yet, but for value changes, it would be nice to put the old values in a comment above/before the new values.

Reply:
Nice idea @rgknox . I've added old values as comments now.